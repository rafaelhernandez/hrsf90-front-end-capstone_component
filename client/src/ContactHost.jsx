import React from 'react';

const ContactHost = (props) => (
  <div className="div-description-contact-host">
    <a href="#">Contact Host</a>
  </div>
);
export default ContactHost;
