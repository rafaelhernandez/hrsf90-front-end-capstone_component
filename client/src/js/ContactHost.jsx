import React from 'react';

const ContactHost = (props) => (
  <div className="div-description-contact-host">
    <p>
      <a className="a-description-fold" href="#">Contact Host</a>
    </p>
  </div>
);
export default ContactHost;
