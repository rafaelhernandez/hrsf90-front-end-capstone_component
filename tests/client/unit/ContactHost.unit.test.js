import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import ContactHost from '../../../client/src/ContactHost.jsx';
import descriptionSample from '../../../db/data/description_sample.js';

configure({ adapter: new Adapter() });

describe('ContactHost', () => {
  it('should render correctly', () => {
    const output = shallow(
      <ContactHost></ContactHost>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});
