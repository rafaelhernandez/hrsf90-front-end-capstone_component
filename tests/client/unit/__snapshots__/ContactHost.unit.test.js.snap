// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactHost should render correctly 1`] = `
<div
  className="div-description-contact-host"
>
  <a
    href="#"
  >
    Contact Host
  </a>
</div>
`;
